pkg/cluster/internal/create/create.go

```diff
@@ -32,20 +32,23 @@ import (
 	"sigs.k8s.io/kind/pkg/cluster/internal/delete"
 	logutil "sigs.k8s.io/kind/pkg/log"
 
+	pkgutil "k8s.io/minikube/pkg/util"
```

Comment:
i think we should use client-go instead of minikube utils here.

Reply:
I will see what I can do. I thought that could be overkill to add another package just for that, but that simplified the config and context management a lot.

Reply:
i think minikube would be a new package here.

client-go is already used in kind:
https://github.com/kubernetes-sigs/kind/blob/master/go.mod#L31

Reply:
yes, it would be. Frankly, couldn't find yet a more straightforward and clean way of setting up  kubeconfig as the way minikube util does. Would it be a disaster to you if I add minikube as package here?

Reply:
my preferred vote would be to not add the minikube package, but i would defer to @BenTheElder for a comment too.

Reply:
No worries, I will take care of it and not add the package. Give me sometime to come up with something else. ;-)

Reply:
@neolit123 I have refactored to remove the `minikube` dependency. It makes sense as `minikube` would pull too many indirect dependencies. I plan to do better as I can also read kind's config into a *api.config, but I would like to work on it in a separate PR.

Reply:
minikube per se is probably fine, but I do prefer the minimal dependencies were we can get the same functionality.